.golangci.yml

```diff
@@ -3,11 +3,11 @@ run:
   go: 1.18
 linters:
```

Comment:
After reviewing the options added by `errcheck`, I think we should be also add the following setting.
@abs007 could you add this new section?

```suggestion
# All possible options can be found here https://github.com/golangci/golangci-lint/blob/master/.golangci.reference.yml
linters-settings:
  errcheck:
    # Report about not checking of errors in type assertions: `a := b.(MyStruct)`.
    # Such cases aren't reported by default.
    # Default: false
    check-type-assertions: true
linters:
```